screens/ShoppingList.js

```diff
@@ -0,0 +1,114 @@
+
+import React, { Component } from 'react';
+import { StyleSheet, Text, View, Dimensions, Image, AppRegistry } from 'react-native';
+
+export default class ShoppingList extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={styles.orangeBox}>
+          <View style={styles.formatInOrangeBox}>
+            <Image source ={require('./../assets/images/shopping-cart.png')} style={styles.ShoppingCart} />
+            <Image source ={require('./../assets/images/person.png')} style={styles.person} />  
+          </View>
+          <View style={styles.formatInOrangeBox}>
+            <Text style = {styles.header}>Shopping List</Text>
+          </View>
+        </View>
+        <View style={{flex: 1, justifyContent: 'flex-end',  alignItems: 'center'}}>
+          <View style={{width: Dimensions.get("screen").width*.90, height: Dimensions.get("screen").height*.17, backgroundColor: '#6DC4E0', marginBottom: 30}}>
```

Comment:
Move these inline styles out and create a new style. It also looks like these styles are the same as the ones on lines 25 and 31.